go.mod

```diff
@@ -27,6 +27,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require gopkg.in/yaml.v3 v3.0.1
```

Comment:
When this becomes indirect again, please let us revert this change

Reply:
Ok